### ISSUE TITLE
Windows build (workflow) improvement

### ISSUE THREAD

**Issue by papadop** [openmeeg/openmeeg#522]
On windows, the Configure step also includes a lot of package installations which generate quite some logs.
It would be nice if the Configure step only contained the cmake configure log as for other OSes (i.e. split the actual Configure step in two pieces).

**Comment by larsoner** [comment]
I tend to agree, but this seems to be the preferred/modern mode of using vcpkg:

https://vcpkg.readthedocs.io/en/latest/users/manifests/

It is possible in principle to use it in classic mode *first*, but then you have to get all the setup steps correct (compiler version, target, etc.) on the command line call, rather than having them be discovered by `vcpkg` through cmake. Otherwise it just builds them again anyway.

One thing that should help on CIs at least is caching. With proper caching these steps should be no-ops because the libraries should already be there. So it might be nice to first fix the caching on Windows (it's broken for some reason) and then mess with modifying how `vcpkg` is used only if we need to.
